### PR TITLE
[docs](ip function) add ipv4/v6 datatype and some ip functions' docs

### PR DESCRIPTION
**Pull request by sjyango**
## Proposed changes

Issue Number: close #xxx

Add some docs for ipv4/ipv6 datatype and some ip functions, such as:
- ipv4_cidr_to_range
- ipv6_cidr_to_range
- is_ip_address_in_range
- is_ipv4_string
- is_ipv6_string
- to_ipv4
- to_ipv4_or_default
- to_ipv4_or_null
- to_ipv6
- to_ipv6_or_default
- to_ipv6_or_null

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/docs/sql-manual/sql-functions/ip-functions/is-ipv6-string.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+#### Syntax
+
+`BOOLEAN IS_IPV6_STRING(STRING ipv6_str)`
+
+接收一个表示形式为字符串的IPv6地址作为参数，如果为格式正确且合法的IPv6地址，返回true；反之，返回false。
+
+### notice
+
+`如果入参为NULL，则返回NULL，表示无效收入`
```
Comment (HappenLee) [inline docs/zh-CN/docs/sql-manual/sql-functions/ip-functions/is-ipv6-string.md:43]: 收入？输入 please check english doc

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by xinyiZzz — APPROVED** [review]
LGTM

**Comment by HappenLee** [comment]
run buildall

**Pull request merged**